stl/inc/xutility

```diff
@@ -2110,8 +2120,12 @@ namespace ranges {
 
             template <class _Ty>
             _NODISCARD static _CONSTEVAL _Choice_t<_St> _Choose() noexcept {
-                _STL_INTERNAL_CHECK(is_lvalue_reference_v<_Ty>);
+                _STL_INTERNAL_STATIC_ASSERT(is_lvalue_reference_v<_Ty>);
                 if constexpr (is_array_v<remove_reference_t<_Ty>>) {
+                    static_assert(_Has_complete_elements<_Ty>,
+                        "The range access customization point objects "
+                        "std::ranges::begin, std::ranges::end, std::ranges::rbegin, and std::ranges::rend "
+                        "do not accept arrays of incomplete element types.");

```

Comment:
Ungrammarly; shouldn't this be "arrays of incomplete elements" or "arrays with incomplete element type"?